internal/gatewayapi/testdata/securitypolicy-with-authoriztion-client-cidr.in.yaml

```
@@ -13,6 +13,60 @@ gateways:
       allowedRoutes:
         namespaces:
           from: All
+- apiVersion: gateway.networking.k8s.io/v1
+  kind: Gateway
+  metadata:
+    namespace: envoy-gateway
+    name: gateway-2
+  spec:
+    gatewayClassName: envoy-gateway-class
+    listeners:
+    - name: http
+      protocol: HTTP
+      port: 80
+      allowedRoutes:
+        namespaces:
+          from: All
+- apiVersion: gateway.networking.k8s.io/v1
```

what will happen if a Gateway has HTTP and TCP listeners, with HTTPRoutes and TCPRoutes attached to each respectively and a SP attaching to the entire Gateway ? will the SP have an error due to validateSecurityPolicyForTCP ?

---

I had it error, when we had a gateway with http and tcp listeners and was using something not supported with tcp. @kkk777-7 had me remove it, now it will apply for the httproute and silently fail the parts that aren't support with the tpc route